README.md

```diff
@@ -71,7 +102,7 @@ or place the following in the `<head>` tag:
 > You must match the version string below with the exact version of this package used by your application.
 
 ```html
-<link rel="stylesheet" href="https://react.seam.co/v/1.0.0/index.min.css" />
+<link rel="stylesheet" href="https://react.seam.co/v/1.7.0/index.min.css" />
```

Comment:
Is there a way we could automate this versioning? Will we have to update the README every time?

Reply:
Yea, feels bad. We could add some automation via preversion script. Or just be ok that this will not always be the latest version.